src/osm_flex/download.py

```diff
@@ -131,9 +166,4 @@ def get_planet_file(save_path=Path(OSM_DATA_DIR,'planet-latest.osm.pbf'),
     ----------
     save_path : str or pathlib.Path
     """
-
-    if not Path(save_path).is_file() or overwrite:
-        LOGGER.info(f'Downloading file as {save_path}')
-        urllib.request.urlretrieve(PLANET_URL, save_path)
-    else:
-        LOGGER.info(f'file already exists as {save_path}')
+    _download_file(PLANET_URL, save_path, overwrite)
```

Comment:
Is there a reason for not returning the filepath here?

Reply:
This function does not compute one, but takes the exact same path as passed to it. I think it does not make sense for a function to return one of its arguments unchanged

Reply:
I see both arguments. for consistency it could be nice to still return the filepath. maybe. but no strong preference

Reply:
The point is that most people will probably be happy with the default, but the default is hard to obtain otherwise. Thus, returning it would be nice, even though I agree it is kindoff redundant.

Reply:
It is done